rskj-core/src/test/java/co/rsk/blockchain/utils/BlockGenerator.java

```diff
@@ -297,8 +301,16 @@ public Block createChildBlock(Block parent, List<Transaction> txs, List<BlockHea
     }
 
     public Block createBlock(int number, int ntxs) {
-        Bloom logBloom = new Bloom();
-        Block parent = getGenesisBlock();
+        return createBlock(number, ntxs, -1);
+    }
+
+    public Block createBlock(int number, int ntxs, long gasLimit){
```

Comment:
Are we going to allow this new `createBlock` method to be called outside of `BlockGenerator`? If that's the case, I would add another separate test for this instead of updating the old one. If not, then I would change its visibility to `private`, and would also add another test, so we're sure that we are testing the `gasLimit` being `-1` and `!-1`.

Reply:
36363e3